dev-requirements.txt

```diff
@@ -26,8 +26,10 @@ PyVCF==0.6.7
 freezegun==0.3.6
 guppy==0.1.10
 snakefood==1.4
+lxml==3.4.4
 Sphinx==1.4.6
 sphinx_rtd_theme
+sphinx-argparse==0.1.15
```

Comment:
Do you think that should be in requirements? I consider the doc creation as the development dependencies. We can manage RTD build dependencies in a nicer way using the `environment.yml`. However, this PR is untested on RTD.